main.pas

```diff
@@ -2386,6 +2397,19 @@ procedure TForm1.Load1Click(Sender: TObject);
     try
       for x := 0 to 1000 do
         datablock[x] := -1;
+
+      // Load STR/HEX label data, if it exists
+      labelseg := seg[1] + (seg[2] - seg[1]);
+      x := 0;
+      while (labelseg + x + 4) < qstfile[f].size do
```

Comment:
clever!
saving extra data like this.

Just make sure to not over use this for other things since it will make quest file bigger. Max file size compressed is 32k